config.json

```diff
@@ -408,7 +408,38 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
+  "key_features": [
+    {
+      "title": "Simple",
+      "content": "Zig lets you focus on debugging your application rather than debugging your programming language knowledge.",
```

Comment:
This line exceeds the maximum length of 100 by 7 characters. Could you shorten it?

Reply:
```suggestion
      "content": "Zig lets you focus on debugging your application rather than your programming language knowledge.",
```